### PR TITLE
Pull request to merge to development

### PR DESCRIPTION
**Pull request by CalyCherkaoui**
This is part of the Forms Project in The Odin Project’s Ruby on Rails Curriculum.
We created a User model and we added all the functionalities like creating it, editing it.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```
Comment (mosesogwo) [inline app/controllers/users_controller.rb]: Remove commented code.

File: app/views/users/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<h1>USER Sign-up form:</h1>
+
+
+<!-- <form action="/users" method="post" accept-charset="UTF-8">
```
Comment (mosesogwo) [inline app/views/users/edit.html.erb]: Use the **_users\_path_** helper instead of "/users"

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hello!:wave: Nice work :+1: Just a few changes and you can :rocket:**

:point_right: You should exclude routes that are not in use from config/routes.rb. 



:point_right: For UsersController update action, if updating succeeds, redirect to edit_user_path. If it fails, render :edit.

:point_right: See other comments on your code below.

**Optional:** You can create a partial for the user form and use this partial in your new and edit views.

_**Kindly make these changes and submit another code review request.**_

**Any questions? You can reach me on Slack (Moses Ogwo)**

**Cheers:stars:**

### :exclamation:Status: Changes Requested.

File: app/views/users/edit.html.erb (hunk 2 above)
Comment (asiimwedismas) [reply app/views/users/edit.html.erb]: Not need when creating pure html for as per the instructios

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (asiimwedismas) [reply app/controllers/users_controller.rb]: Call commented code has been removed

**Review by rOluochKe — APPROVED** [review]
Hello @CalyCherkaoui and @asiimwedismas 
### You finally got it right, well done :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

### Improvements
- You can make the app view better by using bootstrap

**Pull request merged**